extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,45 +18,42 @@
   #
   # Contributor(s): Stefan Seifert <nine@detonation.org>
   #                 Gervase Markham <gerv@gerv.net>
+  #                 Emma Humphries <ech@emmah.net>
   #%]
 
 [% PROCESS global/header.html.tmpl 
    title = "Bugzilla Etiquette" 
    style = "li { margin: 5px } .heading { font-weight: bold }" %]
 
 <p>
-  There's a number of <i lang="fr">faux pas</i> you can commit when using 
-  [%+ terms.Bugzilla %]. At the very   
-  least, these will make Mozilla contributors upset at you; if committed enough    
-  times they will cause those contributors to demand the disabling of your
-  [%+ terms.Bugzilla %] account. So, ignore this advice at your peril. 
+It is our intention that [% terms.Bugzilla %] remain a useful tool for reporting 
```

Comment:
"remain" is correct.
It is our intention that "the thing" remain a useful tool ...
Wishes in English are expressed in the rarely used subjunctive mood (https://en.wikipedia.org/wiki/English_subjunctive), example given there: that he/she/it own